tools/sync-module-dependencies/internal/apply-replaces.go

```diff
@@ -0,0 +1,124 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/grafana/replace-generator/internal/helpers"
+	"github.com/grafana/replace-generator/internal/types"
+)
+
+// detectLineEnding detects the line ending style used in the content.
+// Returns "\r\n" for Windows-style, "\n" for Unix-style, or "\n" as default.
+func detectLineEnding(content string) string {
+	if strings.Contains(content, "\r\n") {
+		return "\r\n"
+	}
+	return "\n"
+}
+
+func main() {
```

Comment:
We have several main functions declared in one package, could we avoid this? most IDEs will complain and some `go` commands that operate on entire package will fail I think

Reply:
One good alternative could be to use Cobra to create a tool with a few subcommands.